client/me/connected-application-item/index.jsx

```diff
@@ -140,7 +140,7 @@ class ConnectedApplicationItem extends React.Component {
 
 		return (
 			<div>
-				<h2>{ this.props.translate( 'Application Website' ) }</h2>
+				<h2>{ this.props.translate( 'Application website' ) }</h2>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.